yt/yt/library/process/process.cpp

```diff
@@ -144,6 +144,15 @@ TErrorOr<TString> ResolveBinaryPath(const TString& binary)
     return failure();
 }
 
+std::vector<TString> GetEnviron()
+{
+    std::vector<TString> env;
```

Comment:
reserve?

Reply:
This is marked resolved but was not fixed.

Reply:
It's fixed below https://github.com/ytsaurus/ytsaurus/pull/390/files#diff-b254d392f69ccc1e18b0111c6416b3faaf71ce3dcf83c3aff9a3a8a4adccec47R154